MovieGPT-api/src/main/java/lt/techin/group/project/controller/GenreController.java

```diff
@@ -49,9 +49,9 @@ public ResponseEntity<GenreDto> addGenre(@Valid @RequestBody GenreDto genreDto)
         return ResponseEntity.status(HttpStatus.CREATED).body(newGenre);
     }
 
-    @PutMapping
-    public ResponseEntity<GenreDto> updateGenre(@Valid @RequestBody GenreDto genreDto) {
-        GenreDto newGenre = genreService.updateGenre(genreDto);
+    @PutMapping("/{id}")
+    public ResponseEntity<GenreDto> updateGenre(@Valid @PathVariable Long id, @RequestBody GenreDto genreDto) {
```

Comment:
There is not much point maybe to request GenreDto object as without id it is simply containing its name (String)?